test-requirements.txt

```diff
@@ -1 +1,2 @@
 odoo_test_helper
+odoo-addon-sale_fixed_discount @ git+https://github.com/OCA/account-invoicing.git@refs/pull/1488/head#subdirectory=setup/sale_fixed_discount
```

Comment:
```suggestion
odoo-addon-account-invoice-fixed-discount @ git+https://github.com/OCA/account-invoicing.git@refs/pull/1488/head#subdirectory=setup/account_invoice_fixed_discount
```

@PieterPaulussen This line is intended to install the module that is not yet merged. So, account_invoice_fixed_discount.